README.md

```diff
@@ -22,6 +27,44 @@ git clone https://github.com/IamYunusAli/portfolio.git
 
 open it on editor and enjoy
 
+**Getting stared with learning full stack web developments.**
+
+> Installing used linter dependencies.
+
+### [Webhint](https://webhint.io/)
+
+A customizable linting tool that helps you improve your site's accessibility, speed, cross-browser compatibility, and more by checking your code for best practices and common errors.
+
+1. Run
+   ```
+   npm install --save-dev hint@6.x
+   ```
+   _not sure how to use npm? Read [this](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm)._
+2. **Do not make any changes in config files - they represent style guidelines that you share with your team - which is a group of all Microverse students.**
+   - If you think that change is necessary - open a [Pull Request in this repository](../README.md#contributing) and let your code reviewer know about it.
+3. Run
+   ```
+   npx hint .
+   ```
+4. Fix validation errors.
+
+### [Stylelint](https://stylelint.io/)
+
+A mighty, modern linter that helps you avoid errors and enforce conventions in your styles.
+
+1. Run
+
+   ```
+   npm install --save-dev stylelint@13.x stylelint-scss@3.x stylelint-config-standard@21.x stylelint-csstree-validator@1.x
+   ```
+
+   _not sure how to use npm? Read [this](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm)._
+
+2. **Do not make any changes in config files - they represent style guidelines that you share with your team - which is a group of all Microverse students.**
+   - If you think that change is necessary - open a [Pull Request in this repository](../README.md#contributing) and let your code reviewer know about it.
+3. Run `npx stylelint "**/*.{css,scss}"` on the root of your directory of your project.
+4. Fix linter errors.
+
```

Comment:
Please remove this section, because this is an instruction from Microverse to you, not for the user that will use your portfolio